server/Makefile

```diff
@@ -107,8 +107,9 @@ endif
 GOFLAGS ?= $(GOFLAGS:)
 # We need to export GOBIN to allow it to be set
 # for processes spawned from the Makefile
-export GOBIN ?= $(PWD)/bin
-GO ?= go
+# export GOBIN ?= $(PWD)/bin
+export GOBIN ?= bin
```

Comment:
You can actually override GOBIN with an env var from where you are invoking the command.

Reply:
For the record [here](https://github.com/mattermost/test-private-project/actions/runs/13973968058/workflow) is the pipeline I am using to test the build